pkgs/development/tools/apko/default.nix

```diff
@@ -42,11 +42,8 @@ buildGoModule rec {
   '';
 
   checkFlags = [
-    # networking required to fetch alpine-keys
-    # pulled out into a separate library next release
-    "-skip=TestInitDB"
-    # fails to build image on read-only filesystem
-    "-skip=TestPublish"
+    # fails to run on read-only filesystem
+    "-skip=(TestPublish|TestBuild|TestTarFS)"
   ];
```

Comment:
Im surprised this seemed to work in the past, because it wouldn't work when I was bumping the version